### PR TITLE
added get segment size method

### PR DESCRIPTION
**Pull request by AmsTaFFix**

### CODE REVIEW

File: src/Request/GetSegmentSizeGetRequest.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function getUri(): string
+    {
+        return '/Api/GetSegmentSize';
+    }
+}
```
Comment (liderman) [inline src/Request/GetSegmentSizeGetRequest.php]: Add an empty string at the end of file

File: src/Response/GetSegmentSizeGetResponse.php [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+     */
+    public function getCountDate(): \DateTime
+    {
+        if (!$this->isOk()) {
+            throw TryToAccessDataFromErrorResponseException::createFromResponse($this);
+        }
+
+        return new \DateTime(strval($this->getSimpleXml()->xpath('/ApiResponse/Data/CountDate')[0]));
+    }
+}
```
Comment (liderman) [inline src/Response/GetSegmentSizeGetResponse.php]: Add an empty string at the end of file

File: src/Request/GetSegmentSizeGetRequest.php (hunk 1 above)
Comment (AmsTaFFix) [reply src/Request/GetSegmentSizeGetRequest.php]: fixed

File: src/Response/GetSegmentSizeGetResponse.php (hunk 2 above)
Comment (AmsTaFFix) [reply src/Response/GetSegmentSizeGetResponse.php]: fixed

**Pull request merged**